Valet.xcodeproj/project.pbxproj

```diff
@@ -497,6 +639,106 @@
 /* End PBXVariantGroup section */
 
 /* Begin XCBuildConfiguration section */
+		26E682811BA8B3F900EFF4EA /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				CURRENT_PROJECT_VERSION = 1;
```

Comment:
We can leave them in, given that we already had them elsewhere. I can do a pass to remove unused key/values from the project file another time.